examples/jest-vtu-example/package.json

```diff
@@ -31,16 +31,17 @@
       "^~~/(.*)$": "<rootDir>/$1"
     },
     "transform": {
-      "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
-      ".*\\.(vue)$": "<rootDir>/node_modules/vue-jest"
+      "^.+\\.js$": "babel-jest",
+      ".*\\.(vue)$": "vue-jest"
     },
     "snapshotSerializers": [
-      "<rootDir>/node_modules/jest-serializer-vue"
+      "jest-serializer-vue"
     ],
     "collectCoverage": true,
     "collectCoverageFrom": [
       "<rootDir>/components/**/*.vue",
       "<rootDir>/pages/*.vue"
-    ]
+    ],
+    "testURL": "http://localhost/"
```

Comment:
Shouldn't  we specify default listen port here? (http://localhost:3000)

Reply:
apparently not, having just the hostname is enough to remove the `SecurityError: localStorage is not available for opaque origins` error

Reply:
Thanks for the clarification.